packages/react/src/Tooltip/Tooltip.features.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import {Button, Box, Link, Stack} from '..'
+import {Tooltip} from './Tooltip'
+import {BookIcon} from '@primer/octicons-react'
+
+export default {
+  title: 'Components/Tooltip/Features',
+  component: Tooltip,
+} as Meta<typeof Tooltip>
+
+export const AnchorHasMargin = () => (
+  <Box padding="spacious">
+    <Tooltip text="Tooltip is still centered">
+      <Button style={{marginLeft: '16px'}}>Button has 16px margin Left</Button>
```

Comment:
Stray capitalization

```suggestion
<Button style={{marginLeft: '16px'}}>Button has 16px margin left</Button>
```